.yamllint

```diff
@@ -10,4 +10,7 @@ rules:
   brackets:
     max-spaces-inside: 1
     level: error
+  document-start: disable
   line-length: disable
+  key-duplicates: enable
+  truthy: disable
```

Comment:
truthy is debatable too and does not play well with "on" use on travis config, better off than adding exclude for travis file.